### PR TITLE
domains discovered during reinstall macOS 10.14.4

### PR DESCRIPTION
**Pull request by SilverBut**
Those domains are used for multiple purposes when downloading Internet recovery image and reinstalling macOS 10.14.4 on MacBook.

### CODE REVIEW

File: apple.china.conf [hunk 1]
```diff
@@ -42,6 +44,7 @@ server=/images.apple.com/114.114.114.114
 server=/init-p01md.apple.com/114.114.114.114
 server=/init-p01st.push.apple.com/114.114.114.114
 server=/init-s01st.push.apple.com/114.114.114.114
+server=/init.ess.apple.com/114.114.114.114
```
Comment (felixonmars) [inline apple.china.conf]: This domain doesn't seem to have a China mainland node.

File: apple.china.conf [hunk 2]
```diff
@@ -3,9 +3,11 @@ server=/a2.mzstatic.com/114.114.114.114
 server=/a3.mzstatic.com/114.114.114.114
 server=/a4.mzstatic.com/114.114.114.114
 server=/a5.mzstatic.com/114.114.114.114
+server=/aaplimg.com/114.114.114.114
 server=/adcdownload.apple.com/114.114.114.114
 server=/appldnld.apple.com/114.114.114.114
 server=/apps.mzstatic.com/114.114.114.114
+server=/bag.itunes.apple.com/114.114.114.114
```
Comment (felixonmars) [inline apple.china.conf]: `itunes.apple.com` is already present, there is no need to add a subdomain.

File: apple.china.conf [hunk 3]
```diff
@@ -3,9 +3,11 @@ server=/a2.mzstatic.com/114.114.114.114
 server=/a3.mzstatic.com/114.114.114.114
 server=/a4.mzstatic.com/114.114.114.114
 server=/a5.mzstatic.com/114.114.114.114
+server=/aaplimg.com/114.114.114.114
```
Comment (felixonmars) [inline apple.china.conf]: This rule seems too broad, what's the pattern under this domain? I would assume something like `itunes-apple.g.aaplimg.com`

File: apple.china.conf (hunk 1 above)
Comment (SilverBut) [reply apple.china.conf]: Just wrote a new script and analyze the pcap file automatically. You are right. This domain shall not be included in the file and I'll remove it later.

File: apple.china.conf (hunk 3 above)
Comment (SilverBut) [reply apple.china.conf]: Two item is added for `aaplimg.com`.

File: apple.china.conf (hunk 2 above)
Comment (SilverBut) [reply apple.china.conf]: Agree!

**Comment by SilverBut** [comment]
Do you believe that some of this pull-request process can be automatically performed by Travis? Maybe few lines of `dig @114.114.114.114`, `dig @8.8.8.8` and a copy of APNIC library is enough. Let me try...

**Comment by felixonmars** [comment]
I actually have verify.py for this, not yet integrated with Travis, though.

**Comment by felixonmars** [comment]
All verified. Many thanks!

**Pull request merged**